corehq/apps/registration/templates/registration/partials/choose_your_plan.html

```diff
@@ -0,0 +1,94 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div id="registration-choose-plan-container"
+     class="reg-form-container reg-plan-container">
+  <div class="row">
+
+    <div class="col-xs-6">
+      <div class="plan-container plan-container-professional">
+        <div class="plan-header">
+          <h2>
+            <small>CommCare</small>
+            Professional
```

Comment:
trans

Reply:
i don't think we're translating these on dimagi.com since it's the name of a product / offering